src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/LSFJobManager.groovy

```diff
@@ -65,32 +70,31 @@ class LSFJobManager extends AbstractLSFJobManager {
      *    * Long (with status flag)
      */
     ZonedDateTime parseTime(String str) {
```

Comment:
In line 46 you could remove the arguments for the `DateTimeHelper` since you don't use them anymore